MLOps/intro-to-ml-model-deployment-and-management/intro-to-ml-model-deployment-and-management.md

```diff
@@ -0,0 +1,811 @@
+# Intro to ML Model Deployment and Management
+
+## Outline
+
+- [Objective](#objective)
+- [Prerequisites](#prerequisites)
+- [Task 1: Explore H2O.ai Platform Studio](#task-1-explore-h2oai-platform-studio)
+- [Task 2: Create Driverless AI Project and Share it with MLOps](#task-2-create-driverless-ai-project-and-share-it-with-mlops)
+- [Task 3: Machine Learning Operations Concepts](#task-3-machine-learning-operations-concepts)
+- [Task 4: Tour of MLOps UI](#task-4-tour-of-mlops-ui)
+- [Task 5: Interactive and Batch Scoring via MLOps Model Deployment](#task-5-interactive-and-batch-scoring-via-mlops-model-deployment)
+- [Task 6: Monitor MLOps Deployed Model](#task-6-monitor-mlops-deployed-model)
+- [Task 7: Challenge](#task-7-challenge)
+- [Next Steps](#next-steps)
+- [Appendix A: AI Glossary](#appendix-a-ai-glossary)
+
```

Comment:
👍 for removing the appendix section that was commented! 👍   There are pros to not having commented code/content part of the checked-in code.  Very clean!